package.json

```diff
@@ -9,8 +9,10 @@
   "description": "A layout building and flashing tool for QMK firmware",
   "repository": "https://github.com/e11i0t23/qmk_configurator_electron.git",
   "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
     "postinstall": "electron-builder install-app-deps",
-    "start": "electron ./src/main",
+    "start": "npm run build && electron ./dist/main.js",
     "pack": "electron-builder --dir",
     "dist": "electron-builder --publish onTagOrDraft"
```

Comment:
Not tested but I think we may need to add `npm run build here` unless electron builder automatically knows to compile typescript files before building